src/BBT.StructureTools.Tests/Convert/Registrations/WholeStructure/TemporalDataHandler.cs

```diff
@@ -0,0 +1,54 @@
+﻿namespace BBT.StructureTools.Tests.Convert.Registrations.WholeStructure
```

Comment:
This isn't part of the structure but based on the test data, move to `BBT.StructureTools.Tests.Convert.TestData` because the `ITemporalData` interface which is relevant for the handler is also located there.

Reply:
done